packages/clay-card/src/Card.tsx

```diff
@@ -35,78 +34,69 @@ interface IProps
 			HTMLAnchorElement | HTMLSpanElement | HTMLDivElement
 		> {}
 
-const ClayCard: React.FunctionComponent<IProps> & {
-	AspectRatio: typeof AspectRatio;
-	Body: typeof Body;
-	Caption: typeof Caption;
-	Description: typeof Description;
-	Group: typeof Group;
-	Row: typeof Row;
-} = ({
+const ClayCard: React.FunctionComponent<IProps> = ({
 	children,
 	className,
 	displayType,
-	horizontal,
-	href,
-	interactive,
-	onClick,
 	selectable = false,
 	...otherProps
-}: IProps) => {
+}) => {
```

Comment:
I think the `IProps` is casted here for our docs site. Could you confirm whether that API table still picks up this component without the type?

Reply:
Hmm I'm going to take a look, I think it was to take the default export that is being `ClayCardHybrid` now. I'll check what's going on.